### PR TITLE
update look and feel version to 1.19.4

### PR DESCRIPTION
**Pull request by borislavpetrovhmcts**
Update look and feel version to 1.19.4

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by tchow8** [comment]
Yarn lock update?

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "description": "Testing framwork for @hmcts/one-per-page",
   "homepage": "https://github.com/hmcts/one-per-page-test-suite#readme",
   "main": "./index.js",
-  "version": "2.0.17",
+  "version": "2.0.18",
```
Comment (hosainnet) [inline package.json:6]: Should this be 2.1.0 to reflect that we upgraded a 1.x.x dependency?

Comment (tchow8) [reply package.json]: Boris pointed out to me that we use ^ for getting versions in the other repos.
Meaning only patch is auto updated it seems. I don't think this update of look-and-feel significantly changes things, and would be useful for all out apps to automatically pull I guess?
In this case lets leave as patch?

Comment (tchow8) [reply package.json:6]: For my earlier comment of ^ updating only patch,
Just double checked and it'll update the up to the left most non zero number.
https://devhints.io/semver
But either way is fine I'd say.

**Review by tchow8 — APPROVED** [review]

**Pull request merged**